Xwt.XamMac/Xwt.Mac/TreeViewBackend.cs

```diff
@@ -192,6 +192,30 @@ public void ExpandToRow (TreePosition pos)
 				p = source.GetParent (p);
 			}
 		}
+
+		public TreePosition GetRowAtPosition (Point p)
+		{
+			var row = Table.GetRow (new System.Drawing.PointF ((float)p.X, (float)p.Y));
+			return row >= 0 ? ((TreeItem)Tree.ItemAtRow (row)).Position : null;
+		}
+
+		public Rectangle GetCellBounds (TreePosition pos, CellView cell, bool includeMargin)
+		{
+			var it = tsource.GetItem (pos);
+			if (it == null)
+				return Rectangle.Zero;
+			var row = Tree.RowForItem (it);
```

Comment:
The result of Tree.RowForItem has to be casted to int, since it returns an nint on XamMac.